lib/product-import.coffee

```diff
@@ -136,18 +136,32 @@ class ProductImport
     posts = _.map productsToProcess, (prodToProcess) =>
       existingProduct = @_isExistingEntry(prodToProcess, existingProducts)
       if existingProduct?
-        @_prepareUpdateProduct(prodToProcess, existingProduct).then (preparedProduct) =>
-          synced = @sync.buildActions(preparedProduct, existingProduct)
-          if synced.shouldUpdate()
-            @client.products.byId(synced.getUpdateId()).update(synced.getUpdatePayload())
-          else
-            Promise.resolve statusCode: 304
+        @_fetchSameForAllAttributesOfProductType(prodToProcess.productType)
+        .then (sameForAllAttributes) =>
+          @_prepareUpdateProduct(prodToProcess, existingProduct).then (preparedProduct) =>
```

Comment:
I do not... thats what we discussed yesterday... the sync module will generate sameForAll update actions every time it encounters an attribute in a variant, mentioned in the list of attribute names passed to the build actions method.
We decided to let it be as this is a special case for carhartt only... the only drawback being more requests per product.